examples/interop/ios-uikit-in-compose/shared/src/iosMain/kotlin/UseUITextField.kt

```diff
@@ -37,7 +37,7 @@ fun UseUITextField() {
             )
             textField
         },
-        modifier = Modifier.padding(4.dp).fillMaxWidth().height(30.dp).border(2.dp, Color.Blue),
+        modifier = Modifier.fillMaxWidth().height(30.dp),
```

Comment:
```suggestion
        modifier = modifier.fillMaxWidth().height(30.dp),
```
We should add `modifier: Modifier = Modfier` to arguments as well

Reply:
Done